gamla/dict_utils.py

```diff
@@ -6,20 +6,46 @@
 
 
 def itemgetter(attr):
+    """
+    Access a dictionary item by its key `attr`.
```

Comment:
Prefer words that are unambiguous and easily read, e.g.
access -> get/retrieve
item -> entry (more common)

Use third person (e.g. "Gets... computes...")